Vizzuality/heco-invest
# feat: Enqueue Klab jobs when projects geometry is changed and KLAB is enabled

This PR allows to enqueue Klab jobs when project geometry or impact areas changes. To do so, new env variable called `KLAB_ENABLED` needs to be set to true.

I am also adding all new required columns to Project model (please see migrations) and basic structure of Klab jobs -- these are more or less dummy now and I will code rest of the stuff when I start working on appropriate tickets.

## Testing instructions

Set KLAB_ENABLED to true and verify that `Klab::SubmitContextsJob` and `Klab::CalculateImpactsJob` are enqueued.

## Tracking

https://vizzuality.atlassian.net/browse/LET-1351
https://vizzuality.atlassian.net/browse/LET-1356
